packages/utils/test/crypto/ec-utils.test.ts

```diff
@@ -159,6 +161,19 @@ describe('Utils/EcUtils', () => {
         ),
       ).rejects.toThrowError('The encrypted data is not well formatted');
     });
+
+    it('should be compatible with previous implementation of eccrypto', async () => {
+      const dataTypes = { native: eccryptoNativeData, browser: eccryptoBrowserData } as const;
+      for (const dataType in dataTypes) {
```

Comment:
use it.each?

Reply:
done in https://github.com/RequestNetwork/requestNetwork/pull/1229/commits/9f792560b62e59850390b1c4011ad1c5a89a87d4